ruby_on_rails/environment_protection.md

```diff
@@ -1,8 +1,40 @@
 # Staging Environment Protection
 
-HTTP Basic Authentication should be configured to prevent public traffic on our develop applications
+## Configuration for Deploio
 
-To setup authentication, configure the application controller like that:
+HTTP Basic Authentication should be configured to prevent public traffic on our development applications.
+
+With Deploio, you have two options:
+
+1. **Configuring Basic Auth on the side of Deploio**
+    - When using Deploio Basic Auth, you can't set the Basic Auth credentials manually. You can only regenerate them by disabling and reenabling Basic Auth for a given project or application.
+    - Basic Auth is not controlled via environment variables.
```

Comment:
I'd not do that. I want to keep this on the app layer, so I bring it with me when moving the app